.github/workflows/velox_docker.yml

```diff
@@ -510,6 +533,8 @@ jobs:
       - name: Setup java and maven
         run: |
           apt-get update && apt-get install -y openjdk-8-jdk maven wget
+          apt remove openjdk-11* -y
+          export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
```

Comment:
GHA creates new shell for each step so this export command may not work. You can try using `>> $GITHUB_ENV` to store the variables.

Reply:
Thanks, fixed!!